### PR TITLE
feat(perf_issues): Update new perf group limit from 60 to 5

### PR DESCRIPTION
**Pull request by udameli**

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -116,7 +116,7 @@
 issue_rate_limiter = RedisSlidingWindowRateLimiter(
     **settings.SENTRY_PERFORMANCE_ISSUES_RATE_LIMITER_OPTIONS
 )
-PERFORMANCE_ISSUE_QUOTA = Quota(3600, 60, 60)
+PERFORMANCE_ISSUE_QUOTA = Quota(3600, 60, 5)
```
Comment (udameli) [inline src/sentry/event_manager.py:119]: happy to bump it up if 5 is too low

**Review by k-fish — APPROVED** [review]
Looks good for now, we can always bump it later

**Review by wedamija — APPROVED** [review]
Looks good to me as a temporary solution

**Review by gggritso — APPROVED** [review]
Fast 5️⃣

**Pull request merged**